src/main/components/form/validator.ts

```diff
@@ -63,15 +62,25 @@ export const isDateInputInvalid: DateValidator = (date: CaseDate | undefined) =>
   const month = parseInt(date.month, 10) || 0;
   const day = parseInt(date.day, 10) || 0;
 
-  if (month < 1 || month > 12 || day < 1 || day > 31) {
+  if (day < 1 || day > 31) {
```

Comment:
Maybe we need a validation for non-leap years? For example person will enter February 29, and it will be valid in non-leap year? Just a suggestion/question do we need this.

Reply:
It's actually not necessary. The code will try to create a date object from the input provided. If user has entered a date like 29th Feb in a non leap year it will fail to create the date object and will return an error to say its a invalid date.

Reply:
I was going to add a test for you but its already there :)